README.md

```diff
@@ -6,13 +6,17 @@ An HTTP service for the controlling of search API
 
 * Run `make debug`
 
-In order to see new search in the UI:
-* run the [dp-design-system](https://github.com/ONSdigital/dp-design-system)
+Run new search in the UI:
+* run the [web journey](https://github.com/ONSdigital/dp/blob/main/guides/INSTALLING.md#web-journey)
 * set `SearchRoutesEnabled` to `true` in the config of the [dp-frontend-router](https://github.com/ONSdigital/dp-frontend-router)
```

Comment:
It might be worth mentioning the A/B testing variable (default sets 10% of users to use new search and 90% to old search). Configuration value is - `SearchABTestPercentage` in dp-frontend-router - set this to 100 to force all requests to new search.

```suggestion
* set `SearchRoutesEnabled` to `true` and `SearchABTestPercentage` to `100` in the config of the [dp-frontend-router](https://github.com/ONSdigital/dp-frontend-router)
```